docs/rule-updates.mdx

```diff
@@ -12,6 +12,116 @@ import MoreHelp from "/src/components/MoreHelp"
 
 Welcome to monthly rule updates! This document includes selected new rules, removed false positives (FP), and other rule updates. These new rules and their updates are made by the Semgrep community and r2c.
 
+## August 2022
```

Comment:
@minusworld and @raghavjain3 please check whether the structure of this update looks fine - feel free to suggest edits or commit anything you need.

Reply:
Seems OK to me. Thanks! :D